### PR TITLE
Add and update the documentation for test_builder

### PR DESCRIPTION
**Pull request by oserres**
Add and update the documentation for the new test_builder.

### CODE REVIEW

File: workloads/README.md [hunk 1]
```diff
@@ -88,61 +88,101 @@ bazel run :distbench -c opt -- node_manager --test_sequencer=localhost:10000 --p
...
+Instance summary:
+  client/0: Tx: 0.9 MiB/s, Rx:1178.8 MiB/s
+  server/0: Tx: 1178.8 MiB/s, Rx:0.9 MiB/s
+Global summary:
+  Total time: 30.012s
+  Total Tx: 35403 MiB (1179.6 MiB/s), Total Nb RPCs: 141510 (4.72 kQPS)
+```
+
+In the `result_folder/`, you can find the tesst configuration as well as the
+a binary and comprossed protobuf of the results. In clear text, the protobof
```
Comment (danmanj) [inline workloads/README.md:126]: compressed

Comment (danmanj) [reply workloads/README.md:126]: s/protobof/protobuf

File: workloads/README.md [hunk 2]
```diff
@@ -288,19 +335,29 @@ We target around 100 nodes for this benchmark.
 
 ```yaml
   actions {
-    name: "clique_queries"
+    name: 'clique_queries'
     iterations {
       max_duration_us: 10000000
-      open_loop_interval_ns: 16000000
-      open_loop_interval_distribution: "sync_burst"
+      open_loop_interval_ns: 160000000
```
Comment (danmanj) [inline workloads/README.md:341]: this seems like 1 too many zeros.... did I make a typo?

**Review by danmanj — APPROVED** [review]

File: workloads/README.md (hunk 2 above)
Comment (oserres) [reply workloads/README.md]: No, my bad, fixed.

**Pull request merged**